.changeset/nervous-moles-sleep.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/radio-group': patch
+'@twilio-paste/core': patch
+---
+
+Value is added as a prop used specifically by RadioGroup as it was inheriting its prop interface from InlineContorlGroup which doens't use value at all.
```

Comment:
I started reading the changes by only reading the changesets, to see if I could understand the changes. I had a hard time understanding the changesets so I'm providing some suggested improvements.

```suggestion
[RadioGroup] Added optional `value` string prop which this component needs and is no longer on the extended `InlineControlGroup` interface.
```